src/ui/components/Header/UserOptions.tsx

```diff
@@ -34,10 +43,21 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
   const onSettingsClick = () => {
     setExpanded(false);
     trackEvent("user_options.select_settings");
-
     setModal("settings");
   };
 
+  const onLayoutDockBottomRightClick = () => {
+    dispatch(setToolboxLayout("ide"));
+  };
+
+  const onLayoutDockLeftClick = () => {
+    dispatch(setToolboxLayout("left"));
+  };
+
+  const onLayoutDockBottomClick = () => {
+    dispatch(setToolboxLayout("bottom"));
+  };
```

Comment:
nit: I'd do `const onLayoutChange = (orientation) => dispatch(setToolboxLayout(orientation));` again pretty minor

Reply:
Thanks! Will look into that